src/user-management/interfaces/create-user-options.interface.ts

```diff
@@ -2,7 +2,7 @@ export interface CreateUserOptions {
   email: string;
   password?: string;
   passwordHash?: string;
-  passwordHashType?: 'bcrypt' | 'firebase-scrypt' | 'ssha';
+  passwordHashType?: 'bcrypt' | 'firebase-scrypt' | 'ssha' | 'scrypt';
```

Comment:
Should abstract these hash types into its own type since it's used in several spots?

Reply:
Sounds good.